Form/Type/MultiUploadType.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Sonata\MediaBundle\Form\Type;
+
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolverInterface;
+
+class MultiUploadType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('context', 'hidden', array('data' => $options['context']))
+            ->add('provider', 'hidden', array('data' => $options['provider']))
+            ->add('files', 'file', array(
+                'multiple' => true
+            ))
+        ;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function setDefaultOptions(OptionsResolverInterface $resolver)
+    {
+        $resolver->setDefaults(array(
+            'context' => 'default',
+            'provider' => null
+        ));
+    }
+
+    public function getName()
+    {
+        return 'multi_upload';
+    }
+}
```

Comment:
Please configure your IDE to always add a newline at end of file, this will make the red pictogram disappear.